src/components/Comment/index.spec.tsx

```diff
@@ -469,4 +468,93 @@ describe(__filename, () => {
       );
     });
   });
+
+  describe('discarding a new comment', () => {
+    it('renders a discard button to cancel entering a comment', () => {
+      const versionId = 33;
+      const fileName = 'file.name.json';
+      const line = 100;
+      const store = configureStore();
+      const payload = {
+        fileName,
+        line,
+        versionId,
+      };
```

Comment:
Because all of this setup is exactly the same for all three tests for "discarding a new comment", I would consider moving it up into the `describe` block, rather than repeating it for each test.

Reply:
I repeated it, because `payload` is used in assertion, and I tried to avoid making assertion statement depend on variables out of the `test`. Maybe I can move `fileName`, `line` and `versionId` out to `describe` to remove some duplications, but I am not very sure, since `payload={fileName, line, versionId}`...